index.html

```diff
@@ -331,47 +339,60 @@
             scheduleObj.avatar = "/images/" + scheduleObj.name.toLowerCase().replace(/\s/g, '') + ".jpg";
           }
           scheduleObj.roomLink = "/room/" + scheduleObj.room.toLowerCase().replace(/ /g,"_");
-        /*} else if (type == "teacher") {
-          if (scheduleObj.period <= 8) { //If it's a class period
+        //Add in photos if the teacher's schedule is being rendered
+        } else if (type == "teacher") {
+          console.log("I know that type is teacher");
+          if (letterToNum(scheduleObj.period) <= 8) { //If it's a class period
+            console.log("I know that the period is less than or equal to eight");
             var userOpenPeriod = false; //Find out if the user has a class that period
-            for (var i = 0; i < schedule.length; i++) {
-              if (schedule[i].period == scheduleObj.period && \
-                (schedule[i].name == "Free Period" || \
-                  schedule[i] == "MS Study Hall" || \
-                  schedule[i] == "Guided Study Hall")) { //TODO find the name of GSH
+            for (var i = 0; i < userSchedule.classes.length; i++) {
+              if (userSchedule.classes[i].period == scheduleObj.period && (userSchedule.classes[i].name == "Free Period" || userSchedule.classes[i].name == "MS Study Hall" || userSchedule.classes[i].name == "Guided Study Hall")) { //TODO find the name of GSH
                 userOpenPeriod = true;
                 break;
               }
             }
-
-            if (scheduleObj.name = "Free Period" && userOpenPeriod) {
-              scheduleObj.avatar =
+            if (scheduleObj.name == "Free Period" && userOpenPeriod) {
+              scheduleObj.avatar = "check";
             } else {
-
+              scheduleObj.avatar = "clear";
             }
-          }*/
+          } else {
+            continue; //Skip over lunch, advisory, assembly, etc.
+          }
         }
+
         try {
-            var times = days[0][day]["MS"][currentSlot]["time"].split('-');
-            scheduleObj.startTime = times[0].trim();
-            scheduleObj.endTime = times[1].trim();
-            todaySchedule.push(scheduleObj);
+          console.log("Im being tried!");
+          var times = days[0][day]["MS"][currentSlot]["time"].split('-');
+          scheduleObj.startTime = times[0].trim();
+          scheduleObj.endTime = times[1].trim();
+          todaySchedule.push(scheduleObj);
         } catch (error) {
```

Comment:
This is not recommended, since it eats the exception instead of letting it go to the console. Can we remove the try block instead?